src/instructlab/config/init.py

```diff
@@ -152,6 +152,7 @@ def init(
     if param_source != click.core.ParameterSource.ENVIRONMENT:
         cfg.chat.model = model_path
         cfg.generate.model = model_path
```

Comment:
FWIW - I'm not convinced this one is actually used anywhere?

Reply:
@markmc I have a PR #2011 designated to removing the `generate.model` field since it's redundant with `generate.teacher.model_path`. We're just waiting on this one first since it introduces a test as well.